dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/AlertPluginInstanceServiceImpl.java

```diff
@@ -108,6 +115,20 @@ public Map<String, Object> create(User loginUser, int pluginDefineId, String ins
         int i = alertPluginInstanceMapper.insert(alertPluginInstance);
         if (i > 0) {
             log.info("Create alert plugin instance complete, name:{}", alertPluginInstance.getInstanceName());
+            // global instance will be added into global alert group automatically
+            if (instanceType == AlertPluginInstanceType.GLOBAL) {
+                AlertGroup globalAlertGroup = alertGroupMapper.selectById(2);
```

Comment:
Please don't use magic number 2.

Reply:
extract the magic number 2 into local variable: GLOBAL_ALERT_GROUP_ID.